staker-rewards/rewards-merkle-generator.sh

```diff
@@ -5,26 +5,73 @@ set -e
 LOG_START='\n\e[1;36m' # new line + bold + color
 LOG_END='\n\e[0m' # new line + reset color
 
+NODE_CURRENT_VER="$(node --version)"
+NODE_REQUIRED_VER="v14.3.0"
+ if [ "$(printf '%s\n' "$NODE_REQUIRED_VER" "$NODE_CURRENT_VER" | sort -V | head -n1)" != "$NODE_REQUIRED_VER" ]; 
+ then 
+        echo "Required node version must be at least ${NODE_REQUIRED_VER}" 
+        exit 1
+ fi
+
 WORKDIR=$PWD
 
 printf "${LOG_START}Initializing merkle-distributor submodule...${LOG_END}"
 
 git submodule update --init --recursive --remote --rebase --force
 
-printf "${LOG_START}Installing dependencies...${LOG_END}"
+printf "${LOG_START}Installing dependencies for merkle-distributor...${LOG_END}"
 
 cd "$WORKDIR/merkle-distributor"
 npm i
 
+printf "${LOG_START}Installing dependencies for staker-rewards...${LOG_END}"
+
 cd "$WORKDIR"
 npm i
 
-printf "${LOG_START}Generating merkle output object...${LOG_END}"
+printf "${LOG_START}Processing input parameters...${LOG_END}"
+
+help()
+{
+   echo ""
+   echo "Usage: $0 -h <eth_host> -s <start_interval> -e <end_interval> -r <rewards>"
+   echo -e "\t-h Websocket endpoint of the Ethereum node"
+   echo -e "\t-s Start of the interval passed as UNIX timestamp"
+   echo -e "\t-e End of the interval passed as UNIX timestamp"
+   echo -e "\t-r Total KEEP rewards distributed within the given interval passed as 18-decimals number"
+   exit 1 # Exit script after printing help
```

Comment:
We also need optional parameters for `TENDERLY_PROJECT_URL` and `TENDERLY_ACCESS_TOKEN`: 

https://github.com/keep-network/keep-ecdsa/blob/decaa4abd405e9266ade4596ac239ab7f1101621/staker-rewards/rewards.js#L35-L41

Reply:
Also, if I set an environment variable `rewards.js` uses but that is not listed in flags of `rewards-merkle-generator-sh.`, e.g. `DEBUG = on`, I assume it's also going to work, correct?

Reply:
- Regarding the first comment - done in a03b30
- Yes, if you run ex: 
```
DEBUG="on" ./rewards-merkle-generator.sh -h wss://eth-mainnet.. -s 1605225600 -e 1605430400 -r 3934656000000000000000000
```
in `rewards.js` `process.env.DEBUG` will be set to `on`